### PR TITLE
Dtsbps 446 zipfile status report update

### PR DESCRIPTION
**Pull request by gauravchawla73463**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSBPS-446

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java [hunk 1]
```diff
@@ -87,6 +96,9 @@ public void should_return_data_returned_from_the_service() throws Exception {
             .andExpect(jsonPath("$.envelopes[0].zip_deleted").value(envelopes.get(0).zipDeleted))
             .andExpect(jsonPath("$.envelopes[0].rescan_for").value(envelopes.get(0).rescanFor))
             .andExpect(jsonPath("$.envelopes[0].case_number").value(envelopes.get(0).caseNumber))
+            .andExpect(jsonPath("$.envelopes[0].classification")
+            .value(envelopes.get(0).classification.toString().toUpperCase()))
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java:100]: do you need `toString().toUpperCase()`, can you compare without them?

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/StaleBlobControllerTest.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
 import uk.gov.hmcts.reform.bulkscanprocessor.services.storage.StaleBlobFinder;
 
 import java.time.Instant;
+import java.time.ZoneId;
```
Comment (mdayican) [inline src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/StaleBlobControllerTest.java:12]: why?

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java]: used name() instead now

File: src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/StaleBlobControllerTest.java (hunk 2 above)
Comment (gauravchawla73463) [reply src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/controllers/StaleBlobControllerTest.java]: removed

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/zipfilestatus/ZipFileEnvelope.java [hunk 3]
```diff
@@ -51,7 +58,10 @@ public ZipFileEnvelope(
         String envelopeCcdAction,
         boolean zipDeleted,
         String rescanFor,
+        Classification classification,
+        String jurisdiction,
         String caseNumber,
+
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/zipfilestatus/ZipFileEnvelope.java:64]: can you delete the line ?

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java [hunk 4]
```diff
@@ -87,6 +96,9 @@ public void should_return_data_returned_from_the_service() throws Exception {
             .andExpect(jsonPath("$.envelopes[0].zip_deleted").value(envelopes.get(0).zipDeleted))
             .andExpect(jsonPath("$.envelopes[0].rescan_for").value(envelopes.get(0).rescanFor))
             .andExpect(jsonPath("$.envelopes[0].case_number").value(envelopes.get(0).caseNumber))
+            .andExpect(jsonPath("$.envelopes[0].classification")
+            .value(envelopes.get(0).classification.name()))
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java:100]: if it fits 120 can you make it 1 line

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java [hunk 5]
```diff
@@ -95,6 +107,9 @@ public void should_return_data_returned_from_the_service() throws Exception {
             .andExpect(jsonPath("$.envelopes[1].zip_deleted").value(envelopes.get(1).zipDeleted))
             .andExpect(jsonPath("$.envelopes[1].rescan_for").value(envelopes.get(1).rescanFor))
             .andExpect(jsonPath("$.envelopes[1].case_number").value(envelopes.get(1).caseNumber))
+            .andExpect(jsonPath("$.envelopes[1].classification")
+            .value(envelopes.get(1).classification.name()))
```
Comment (mdayican) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java:111]: can you make it 1 line if it fits 120 char limit

**Review by mdayican — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java [hunk 6]
```diff
@@ -46,6 +51,8 @@ public void should_return_data_returned_from_the_service() throws Exception {
                 "AUTO_ATTACHED_TO_CASE",
                 false,
                 "envelope11.zip",
+                Classification.NEW_APPLICATION,
```
Comment (apereverzin) [inline src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java:54]: Makes sense to use static import

Comment (gauravchawla73463) [reply src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/controller/ZipStatusControllerTest.java]: done

**Review by apereverzin — APPROVED** [review]

**Pull request merged**